index.js

```diff
@@ -34,6 +34,7 @@ const providers = {
 	OobaBooga: require('./providers/oobabooga'),
 	Smol: require('./providers/smol'),
 	HuggingChat: require('./providers/huggingchat'),
+	Phind: require('./providers/phind'),
 	Perplexity: require('./providers/perplexity'),
```

Comment:
rearranging for subjective order of readiness/ease of use/quality

```suggestion
	Perplexity: require('./providers/perplexity'),
	Phind: require('./providers/phind'),
	HuggingChat: require('./providers/huggingchat'),
	OobaBooga: require('./providers/oobabooga'),
	Smol: require('./providers/smol'),
```